### PR TITLE
COMP3710 46179605 JINGXU project

### PR DESCRIPTION
**Pull request by Summeralon**
By using and modifying the Unet model to identify and analyze the data set ISIC, the accuracy of dice is above 0.8.
Student ID:46179605
Student Name:JINGXU

### CODE REVIEW

**Comment by Summeralon** [comment]
Student ID:46179605
Student Name:JINGXU

**Comment by SiyuLiu0329** [comment]
- Commit messages should be more specific
- Code check out for Easy/Normal difficulty - Only training and validation sets, no test set
-  Val dsc > 0.8

Other Feedback 
- Please group the `.png` files and put them in a sub-directory
- Please follow python file naming convention e.g. `lorem_ipsum.py`, `()` are generally not accepted

**Comment by rexsw** [comment]
Code looks good for normal difficulty

Lots of commit messages didn’t explain too much about what was in the commit and needed more detail 

Good comments 

Tensorflow has functions for reading and resizing images so there’s no need for cv2. Similarly for sklearn and train test splitting. You don’t want your project to depend on lots of different libraries if it doesn’t have too

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme sparse
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Partially Successful Pull Request (Working Algorithm Delivered on Time in Incorrect Branch) 
**Feedback required. Move image files to sub directory, update the branch to topic-recognition.**
No Proper Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.
